src/components/common/form/error-summary/ErrorSummary.jsx

```diff
@@ -2,6 +2,22 @@ import PropTypes from 'prop-types';
 import { HashLink } from 'react-router-hash-link';
 
 const ErrorSummary = ({ errors }) => {
+  const callbackRef = (summaryErrorList) => {
+    // if node exists (there is a summary error), scroll to the first summary error and focus it
+    if (summaryErrorList) {
+      const firstSummaryError =
+        summaryErrorList.children['summary-error-0'].children[
```

Comment:
Looks as though `summary-error-` is repeated here and in the "html" below, so might be tempted to add that to a variable

Just a thought, could children dot children be something like the below to make it a little clearer:
`getElementById(variableName).firstChild()`

Reply:
I thought of doing it by firstChild, but explicitly wanted to call out the child just in case anything chaned in the future and the first child wasn't the first error, I can change it if that would be preferred